gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/testing/LocalResourceManagerHelper.java

```diff
@@ -186,6 +212,30 @@ public void handle(HttpExchange exchange) {
     }
   }
 
+  private Response handlePost(HttpExchange exchange, String path) throws IOException {
+    String requestBody = decodeContent(exchange.getRequestHeaders(), exchange.getRequestBody());
+    if (!path.contains(":")) {
+      return create(jsonFactory.fromString(requestBody, Project.class));
+    } else {
+      switch (path.split(":", 2)[1]) {
+        case "undelete":
+          return undelete(projectIdFromUri(path));
+        case "getIamPolicy":
+          return getPolicy(projectIdFromUri(path));
+        case "setIamPolicy":
+          return replacePolicy(projectIdFromUri(path),
+              jsonFactory.fromString(requestBody, SetIamPolicyRequest.class).getPolicy());
+        case "testIamPermissions":
+          return testPermissions(projectIdFromUri(path),
+              jsonFactory.fromString(requestBody, TestIamPermissionsRequest.class)
+                  .getPermissions());
+        default:
+          return Error.BAD_REQUEST.response(
+              "The server could not understand the following request URI: POST " + path);
+      }
+    }
+  }
+
```

Comment:
Unrelated comment. can you replace `Integer.valueOf` with `Integer.parseInt` (line 312)?